client/js/nonprofits/donate/page.js

```diff
@@ -16,9 +16,7 @@ const patch = snabbdom.init([
 , require('snabbdom/modules/style')
 ])
 
-const params = url.parse(location.href, true).query
-params.hide_cover_fees_option = params.hide_cover_fees_option || app.hide_cover_fees_option
-params.manual_cover_fees = params.manual_cover_fees || app.manual_cover_fees
+const params = parseDonateParams(document.location, app);
```

Comment:
All these “params” are a little confusing to follow. Nit: maybe rename them to be more descriptive.

Reply:
We can handle that in a follow-on PR. What in particular is an issue?